content/soarian/r4/base/individuals/patient.md

```diff
@@ -0,0 +1,163 @@
+* [Telecom Information](https://hl7.org/fhir/R4/patient-definitions.html#Patient.telecom){:target="_blank"}, may include phone and email
+* [Gender (administrative)](https://hl7.org/fhir/R4/patient-definitions.html#Patient.gender){:target="_blank"}
+* [Extensions including birth sex, ethnicity and race](#extensions){:target="_blank"}
+* [Date of Birth]( https://hl7.org/fhir/R4/patient-definitions.html#Patient.birthDate){:target="_blank"}
+* [Address]( https://hl7.org/fhir/R4/patient-definitions.html#Patient.address){:target="_blank"}
+* [Communication (preferred language)]( https://hl7.org/fhir/R4/patient-definitions.html#Patient.communication){:target="_blank"}
+
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_patient, :r4) %>
+
+## Extensions
+
+* [US Core Race]
+* [US Core Ethnicity]
+* [US Core Birth Sex]
+
+## Search
+
+Search for Patients that meet specified query parameters:
+
+    GET /Patient?:parameters
+
+_Implementation Notes_
+
+* A `422 (unprocessable entity)` HTTP response status code and OperationOutcome are returned when >499 patients qualify for the search criteria. FHIR<sup>®</sup> applications may prompt the user for more specific search criteria to avoid this possibility.
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true, system: true) %>
+
+### Parameters
+
+ Name                 | Required?                                    | Type       | Description
+----------------------|----------------------------------------------|------------|--------------------------------------------------------------------------
+ `_id`                | This or any other required search parameter | [`token`]  | The logical resource ID associated with the resource.
+ `identifier`         | This or any other search parameter, or `_id` | [`token`]  | A patient identifier. Only MR, PI, or SB identifiers are supported and therefore searched. Example: `urn:oid:1.1.1.1.1.1`\|`1022228`
+ `name`               | This or any other search parameter, or `_id` | [`string`] | The start of either the family or the given name of the patient. Previous name values are not searchable. Example: `Doe`
+ `family`             | This or any other search parameter, or `_id` | [`string`] | The start of the patient's family name. Previous name values are not searchable. Example: `Doe`
+ `given`              | This or any other search parameter, or `_id` | [`string`] | The start of the patient's given name. Previous name values are not searchable. Example: `Jane`
+ `birthdate`          | This or any other search parameter, or `_id` | [`date`]   | The patient's date of birth.  Example: `1990-01-01`
+ `gender`             | No                                           | [`token`]  | The patient's gender. Example: `male`
+ `_revinclude`        | No | [`_revinclude`]  | A request to include any Provenance resource in the bundle that refers to a Patient resource in the search results. Only supported with Provenance.
```

Comment:
Does this require additional Notes? Usually while doing revinclude, explicit Provenance scope should also needed.
A similar example from Millennium R4 AllergyIntolerance API
<img width="740" alt="Screenshot 2021-09-17 at 3 08 57 PM" src="https://user-images.githubusercontent.com/6142355/133761858-e2ace8f8-6699-459d-afd7-59ebc5db3281.png">
:

Reply:
@slvsridhar Just to clarify, Soarian Clinicals R4 implementation does support patient/Provenance.read scope and _revinclude can be utilized for patient persona as well.

Reply:
yes @ashu-bdvt . thanks for bringing it up. we have added notes for '_revinclude' in patient resource.